dashboard/src/components/EncryptionKey.jsx

```diff
@@ -48,6 +45,10 @@ const EncryptionKey = ({ isMain }) => {
       await new Promise((resolve) => setTimeout(resolve, 100));
       if (!values.encryptionKey) return toast.error("La clé est obligatoire");
       if (!values.encryptionKeyConfirm) return toast.error("La validation de la clé est obligatoire");
+      if (!import.meta.env.VITE_SKIP_KEY_SECURITY) {
```

Comment:
Ce serait pas plutôt ça qu'on veut mettre ?

```suggestion
      if (!import.meta.env.VITE_TEST_PLAYWRIGHT) {
```